lms/envs/common.py

```diff
@@ -3459,3 +3459,6 @@ def _make_locale_paths(settings):
 from openedx.core.djangoapps.plugins import plugin_apps, plugin_settings, constants as plugin_constants
 INSTALLED_APPS.extend(plugin_apps.get_apps(plugin_constants.ProjectType.LMS))
 plugin_settings.add_plugins(__name__, plugin_constants.ProjectType.LMS, plugin_constants.SettingsType.COMMON)
+
+
+MIDDLEWARE_CLASSES.append('branding_by_referer.SetBrandingByReferer')
```

Comment:
This should not be added here. We will install it with ansible using `EDXAPP_EXTRA_MIDDLEWARE_CLASSES`